test/functional/feature_csv_activation.py

```diff
@@ -83,80 +83,73 @@ def relative_locktime(sdf, srhb, stf, srlb):
-    tx = create_transaction(node, input, address, amount=Decimal("49.98"))
-    tx.nVersion = txversion
-    signtx = sign_transaction(node, tx)
-    signtx.vin[0].scriptSig = CScript([-1, OP_CHECKSEQUENCEVERIFY, OP_DROP] + list(CScript(signtx.vin[0].scriptSig)))
-    return signtx
-
-def create_bip112emptystack(node, input, txversion, address):
-    tx = create_transaction(node, input, address, amount=Decimal("49.98"))
-    tx.nVersion = txversion
-    signtx = sign_transaction(node, tx)
-    signtx.vin[0].scriptSig = CScript([OP_CHECKSEQUENCEVERIFY] + list(CScript(signtx.vin[0].scriptSig)))
-    return signtx
-
-def send_generic_input_tx(node, coinbases, address):
-    return node.sendrawtransaction(ToHex(sign_transaction(node, create_transaction(node, node.getblock(coinbases.pop())['tx'][0], address, amount=Decimal("49.99")))))
-
-def create_bip68txs(node, bip68inputs, txversion, address, locktime_delta=0):
-    """Returns a list of bip68 transactions with different bits set."""
-    txs = []
-    assert len(bip68inputs) >= 16
-    for i, (sdf, srhb, stf, srlb) in enumerate(product(*[[True, False]] * 4)):
-        locktime = relative_locktime(sdf, srhb, stf, srlb)
-        tx = create_transaction(node, bip68inputs[i], address, amount=Decimal("49.98"))
-        tx.nVersion = txversion
-        tx.vin[0].nSequence = locktime + locktime_delta
-        tx = sign_transaction(node, tx)
-        tx.rehash()
-        txs.append({'tx': tx, 'sdf': sdf, 'stf': stf})
-
-    return txs
-
-def create_bip112txs(node, bip112inputs, varyOP_CSV, txversion, address, locktime_delta=0):
-    """Returns a list of bip68 transactions with different bits set."""
-    txs = []
-    assert len(bip112inputs) >= 16
-    for i, (sdf, srhb, stf, srlb) in enumerate(product(*[[True, False]] * 4)):
-        locktime = relative_locktime(sdf, srhb, stf, srlb)
-        tx = create_transaction(node, bip112inputs[i], address, amount=Decimal("49.98"))
-        if (varyOP_CSV):  # if varying OP_CSV, nSequence is fixed
-            tx.vin[0].nSequence = BASE_RELATIVE_LOCKTIME + locktime_delta
-        else:  # vary nSequence instead, OP_CSV is fixed
-            tx.vin[0].nSequence = locktime + locktime_delta
-        tx.nVersion = txversion
-        signtx = sign_transaction(node, tx)
-        if (varyOP_CSV):
-            signtx.vin[0].scriptSig = CScript([locktime, OP_CHECKSEQUENCEVERIFY, OP_DROP] + list(CScript(signtx.vin[0].scriptSig)))
-        else:
-            signtx.vin[0].scriptSig = CScript([BASE_RELATIVE_LOCKTIME, OP_CHECKSEQUENCEVERIFY, OP_DROP] + list(CScript(signtx.vin[0].scriptSig)))
-        tx.rehash()
-        txs.append({'tx': signtx, 'sdf': sdf, 'stf': stf})
-    return txs
 
 class BIP68_112_113Test(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
         self.extra_args = [[
             '-whitelist=noban@127.0.0.1',
-            '-addresstype=legacy',
+            '-acceptnonstdtxn=1',
```

Comment:
Would be nice to not set this. Maybe someone can teach the miniwallet how to create pay-to-raw-pubkey or so?